stack_definition.json

```diff
@@ -212,6 +219,10 @@
           "name": "existing_kms_instance_crn",
           "value": "ref:../2a - Security Service - Key Management/outputs/kms_instance_crn"
         },
+        {
+          "name": "profile_attachments",
+          "value": "ref:../../inputs/profile_attachments"
```

Comment:
Set the default here - point to the AI ICT Guardrails (as opposed to fs cloud) as this is what we're claiming in the stack.